lua/luasnip/loaders/init.lua

```diff
@@ -96,4 +89,15 @@ function M.load_lazy_loaded(fts)
 	end
 end
 
+vim.cmd([[
+		augroup luasnip_watch_reload
+		autocmd BufWritePost * lua require("luasnip.loaders").reload_file(vim.fn.expand("<afile>"))
+		augroup END
+	]])
+function M.reload_file(filename)
```

Comment:
I think the normalization of the path should happen here instead of in each loader module.

Reply:
Mhh yeah, that might be better.
I put them into each loader since it makes for a nicer public interface (whatever kind of path is passed to it, it works), but `reload_file` isn't even public.
I'll make that a bit more obvious (prefix with __ or something), and do the normalization outside those functions. TY👍